app/api/v1/endpoints/strategies.py

```
@@ -582,31 +582,47 @@ async def get_strategy_marketplace(
         window=60,
         user_id=str(current_user.id)
     )
-    
+
     try:
-        # Get marketplace strategies using the new unified service
+        # Import cache manager for performance
+        from app.core.redis import cache_manager
+
+        # Create cache key based on user and filters
+        cache_key = f"marketplace:{str(current_user.id)}:ai_{include_ai}:community_{include_community}"
```

Ruff (0.13.1)

Use explicit conversion flag

Replace with conversion flag

(RUF010)